gallery/src/components/NxSmallTag/NxSmallTagExample.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxSmallTag } from '@sonatype/react-shared-components';
+
+function NxSmallTagExample() {
+  return (
+    <>
```

Comment:
Redundant

Reply:
The unnecessary `<div className='nx-small-tag-container'>` has been removed, so keep fragment here.